MedscribeUI/src/App.tsx

```diff
@@ -4,34 +4,14 @@ import viteLogo from '/vite.svg';
 import { Loader } from '@mantine/core';
 import './App.css';
 
-function App() {
-  const [count, setCount] = useState<number>(0);
+import NoteControlsLayout from './components/NoteControls/NotecontrolsLayout';
 
+function App() {
   return (
     <>
       <div>
-        <Loader color="blue" />;
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button
-          onClick={() => {
-            setCount(count => count + 1);
-          }}
-        >
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
+        <NoteControlsLayout />
```

Comment:
Please just remove everything in this return statement and just put an empty fragment. Lets keep this clean for now if it has no purpose